index.js

```diff
@@ -217,18 +217,20 @@ const handlers = {
 
   "HelpMessageIntent": function () {
     const { slots } = this.event.request.intent;
-    if(slots.MessageType.value === "Find Item Feature") {
-      const speechOutput = findIntentMessage;
-      const reprompt = speechOutput;
-    } else if(slots.MessageType.value === "Store Item Feature") {
-      const speechOutput = storeIntentMessage;
-      const reprompt = speechOutput;
-    } else if(slots.MessageType.value === "List Event Feature") {
-      const speechOutput = listEventMessage;
-      const reprompt = speechOutput;
+    let speechOutput, reprompt;
+    console.log(slots.MessageType.value);
```

Comment:
Oops!!!! @PikkaPikkachu or @shikhar-scs. Remove this

Reply:
Give me final thing. Fast